src/System.Runtime.Serialization.Formatters/ref/System.Runtime.Serialization.Formatters.cs

```diff
@@ -41,7 +70,7 @@ public sealed class OptionalFieldAttribute : Attribute
     {
         public int VersionAdded
         {
-            get { return default(int); }
+            get { throw null; }
```

Comment:
Yeah we just need valid C# so if this is more terse lets use it.  We should probably make a change to GenAPI and standardize across the repo.